src/views/Storybook.js

```diff
@@ -0,0 +1,22 @@
+import React, { PureComponent } from 'react';
+import { connect } from 'react-redux';
+import './Storybook.scss';
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+const modules = ['installation', 'getting-started', 'customize'];
```

Comment:
Interesting, why define an array of 3, and then only use the first element?

Reply:
tbh this was a copy paste from the other PR (and other components) I didn't look into implications